### PR TITLE
Feature - expand DBT support. 

### PR DESCRIPTION
**Pull request by lilyroberts**
Adding suggestions for CLI options with enumerated arg values. Adding build and parse commands.  Adding new flags and arguments to various commands. 

**What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)**
Feature expansion.

**What is the current behavior? (You can also link to an open issue here)**
Limited DBT CLI support.

**What is the new behavior (if this is a feature change)?**
Expanded support for DBT CLI.

### CODE REVIEW

File: src/dbt.ts [hunk 1]
```diff
@@ -3,9 +3,111 @@ const completionSpec: Fig.Spec = {
...
+          name: "--full-refresh",
+          description: "Treat incremental models as table models",
+        },
+        {
+          name: ["-x", "--fail-fast"],
+          description: "Exit immediately if a single model fails to build",
+        },
+        {
+          name: "--profiles-dir",
+          description: "",
```
Comment (QuiiBz) [inline src/dbt.ts:19]: Could you add a description for this option?

File: src/dbt.ts [hunk 2]
```diff
@@ -3,9 +3,111 @@ const completionSpec: Fig.Spec = {
...
+          args: {
+            name: "profiles directory",
```
Comment (QuiiBz) [inline src/dbt.ts:21]: Argument's `name` should ideally have a single word, in this case, `directory`. Could you update this for every argument underneath?

File: src/dbt.ts [hunk 3]
```diff
@@ -3,9 +3,111 @@ const completionSpec: Fig.Spec = {
...
+            description: "Directory containing profiles.yml",
+            template: "folders",
+            suggestions: ["profiles"],
+          },
+        },
+        {
+          name: "--profile",
+          description: "",
```
Comment (QuiiBz) [inline src/dbt.ts:29]: Missing description, looks like it happens a few times underneath

File: src/dbt.ts [hunk 4]
```diff
@@ -3,9 +3,111 @@ const completionSpec: Fig.Spec = {
...
+            isVariadic: true,
+            suggestions: [
+              "path:",
+              "tag:",
+              "config:",
+              "test_type:",
+              "test_name:",
+            ],
+          },
+        },
```
Comment (QuiiBz) [inline src/dbt.ts:89]: We can use [`exclusiveOn`](https://fig.io/docs/reference/option#exclusiveon) for these two options

File: src/dbt.ts [hunk 5]
```diff
@@ -16,11 +118,117 @@ const completionSpec: Fig.Spec = {
...
           description: "Exit immediately if a single model fails to build",
         },
+        {
+          name: "--use-colors",
+          description: "Default value -- colorize run logs",
+        },
+        {
+          name: "--no-use-colors",
+          description: "Disable log colorizing",
+        },
```
Comment (QuiiBz) [inline src/dbt.ts:131]: Same as above for `exclusiveOn`

**Review by QuiiBz — COMMENTED** [review]
Thanks a lot for this first contribution! I left some comments to resolve before merging this. Also, it looks like a lot of options are shared, could you move them to a top-level const to avoid duplicating them?

**Comment by QuiiBz** [comment]
Hi @lilroberts, just pinging you to know if you still want to resolve the code review, or if we should handle it ourselves. Thanks!

**Comment by mschrage** [comment]
Hey @QuiiBz can you make these changes?

**Review by QuiiBz — APPROVED** [review]
Thanks for the initial PR @lilyroberts!

**Pull request merged**